lib/serial_terminal.pm

```diff
@@ -178,7 +178,7 @@ sub upload_file {
     $dst = "ulogs/" . $dst;
     my ($fh, $tmpfilename) = tempfile(UNLINK => 1, SUFFIX => '.openqa.upload');
 
-    die("File $src doesn't exists on SUT") if (script_run("test -f $src", quiet => 1, timeout => $opts{timeout}) != 0);
+    die("File $src does not exist on SUT") if (script_run("test -f $src", quiet => 1, timeout => $opts{timeout}) != 0);
```

Comment:
why you say that it is wrong ? https://www.englisch-hilfen.de/en/grammar/kurzformen.htm short forms is grammatically correct versions of long one

Reply:
the combination of "does not exist**s**" would be wrong. I just fixed it *and* avoided the contraction which is a common good style for formal written english